docs/wallets/cli/dcrd-and-dcrwallet-cli-arguments.md

```diff
@@ -31,12 +32,18 @@ dcrd --logdir=/my/custom/log/directory
             `--testnet`               | Use the test network
             `--simnet`                | Use the simulation test network
             `--regnet`                | Use the regression test network
-    `-d` or `--debuglevel=`           | Logging level for all subsystems {trace, debug, info, warn, error, critical} -- You may also specify <subsystem>=<level>,<subsystem2>=<level>,... to set the log level for individual subsystems -- Use show to list available subsystems (default: info)
+    `-d` or `--debuglevel=`           | Logging level for all subsystems {trace, debug, info, warn, error, critical} -- You may also specify `<subsystem>=<level>`,`<subsystem2>=<level>`,... to set the log level for individual subsystems -- Use `show` to list available subsystems (default: info)
```

Comment:
I think the comma should be within the `code` annotation here, because the comma is part of the correct syntax. For an example `dcrd --debuglevel=RPCS=info,INDX=warn`
```suggestion
    `-d` or `--debuglevel=`           | Logging level for all subsystems {trace, debug, info, warn, error, critical} -- You may also specify `<subsystem>=<level>,<subsystem2>=<level>,...` to set the log level for individual subsystems -- Use `show` to list available subsystems (default: info)
```